src/network/networkd-ndisc.c

```diff
@@ -1419,6 +1436,70 @@ int config_parse_ndisc_deny_listed_prefix(
         }
 }
 
+int config_parse_ndisc_deny_listed_route_prefix(
```

Comment:
Ah, I noticed you copied `config_parse_ndisc_deny_listed_prefix()`. Could you merge the two functions?